spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -13,9 +13,10 @@
     }
     create(:group_assignment, options)
   end
-  let(:grouping) { group_assignment.grouping }
+  let(:grouping)   { group_assignment.grouping }
+  let(:group_name) { "#{Faker::Company.name} Team" }
   let(:group) do
-    group = Group.create(grouping: grouping, title: "#{Faker::Company.name} Team")
+    group = Group.create(grouping: grouping, title: group_name)
```

Comment:
You could possibly get rid of the the `group_name` variable here by defining a new `group` factory in [`factories.rb`](https://github.com/education/classroom/blob/master/spec/factories.rb), something like this:

```
factory :group do
  title { Faker::Company.name }
  slug  { title.parameterize  }
  github_id  { rand(1..1_000_000) }
end
```